pyanaconda/modules/common/task/runnable.py

```diff
@@ -102,16 +110,22 @@ def _task_run_callback(self):
     @async_action_nowait
     def _task_failed_callback(self):
         """Callback for a failed task."""
+        for i in self._failed_signal._methods:
+            log.debug(i)
```

Comment:
The changes in this file should go in, right?
I'd maybe do it in a separate commit, or even PR.

Reply:
Nope, leftover debugging output, oops. :P Should be fixed now. :)